<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PingTraceOptionsWidget</class>
 <widget class="QWidget" name="PingTraceOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ping/Trace</string>
  </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,6,1">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QSlider" name="slidePackets">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSpinBox" name="spinPackets">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64</number>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="slideTimeout">
               <property name="minimum">
                <number>1000</number>
               </property>
               <property name="maximum">
                <number>30000</number>
               </property>
               <property name="singleStep">
                <number>1000</number>
               </property>
               <property name="pageStep">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="lblTimeout">
               <property name="text">
                <string>Timeout</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSpinBox" name="spinTTL">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="lblTTL">
               <property name="text">
                <string>Time to Live</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="slidePacketSize">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>2048</number>
               </property>
               <property name="pageStep">
                <number>16</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSpinBox" name="spinTimeout">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>30000</number>
               </property>
               <property name="singleStep">
                <number>1000</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSlider" name="slideTTL">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="pageStep">
                <number>16</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSlider" name="slideHops">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="lblHops">
               <property name="text">
                <string>Number of Hops</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="spinPacketSize">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>2048</number>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QSpinBox" name="spinHops">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lblPackets">
               <property name="text">
                <string>Packets to Send</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lblPacketSize">
               <property name="text">
                <string>Packet Size</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="lineWidth">
            <number>3</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="cbShowHostDescription">
             <property name="text">
              <string>Show Hosts Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbClearOldInfo">
             <property name="text">
              <string>Clean old info at start</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QCheckBox" name="cbStopTrace">
           <property name="text">
            <string>Stop tracing when a non-responding device is encountered</string>
           </property>
          </widget>
         </item>
        </layout>

 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections>
   <connection>
   <sender>slidePackets</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinPackets</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>657</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinPackets</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slidePackets</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slidePacketSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinPacketSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>559</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinPacketSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slidePacketSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slideTimeout</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinTimeout</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinTimeout</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slideTimeout</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slideTTL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinTTL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinTTL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slideTTL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slideHops</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHops</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHops</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slideHops</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
